server/index.ts

```diff
@@ -108,6 +107,8 @@ app.use((_, res, next) => {
 
 app.use(
 	helmet({
+		// https://github.com/helmetjs/helmet?tab=readme-ov-file#x-powered-by
```

Comment:
We don't need to include links for every single thing here. People can look it up if they want.

```suggestion
```